tutorials/3d/procedural_geometry/arraymesh.rst

```diff
@@ -233,6 +299,74 @@ that you find online.
 
       # Insert committing to the ArrayMesh here.
 
+ .. code-tab:: csharp C#
+
+    public partial class ArrayMeshInstance : MeshInstance3D
```

Comment:
Doc convention would call this class `MyMeshInstance3D`.

Reply:
Updated/fixed -- but can I ask, is there a page that documents these conventions? I see that the [C# style guide](https://docs.godotengine.org/en/stable/tutorials/scripting/c_sharp/c_sharp_style_guide.html) includes several examples with this naming convention, but at the same time, the [documentation writing guidelines](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html) urge us not to use names like `my_var` or `my_func()`.

(I mention this not to argue the point! Just to say that I would find it helpful to have a reference that definitively states what conventions to follow, if one exists.)

Reply:
Most of it is following the [style guide](https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_style_guide.html). For this particular naming thing, I think we discussed it recently in the rocket.chat. I'll take some time to formalize a C# section in the doc writing guidelines page you mentioned 👍